README.md

```diff
@@ -29,10 +29,10 @@ a Lightning node while exposing a robust, language-agnostic API via [Protocol Bu
 🚧 **Work in Progress**:
 - **APIs Under Development**: Expect breaking changes as the project evolves.
 - **Potential Bugs and Inconsistencies**: While progress is being made toward stability, unexpected behavior may occur.
-- **Improved Logging and Error Handling Coming Soon**: Current error handling is rudimentary(specially for CLI), and usability improvements are actively being worked on.
+- **Improved Logging and Error Handling Coming Soon**: Current error handling is rudimentary ( specially for CLI ), and usability improvements are actively being worked on.
```

Comment:
Sorry, meant just add the one space after "rudimentary". The ones inside the parentheses aren't needed.